src/services/spreadsheets/accounts.ts

```diff
@@ -59,33 +54,29 @@ interface Role {
   id: string;
 }
 
-interface Account {
+export interface Account {
   characters: string;
   accountName: string;
   password: string;
   requiredRoles: Role[];
   purpose?: string;
 }
 
-const cache = new LRUCache<string, Account>({
+const credentialsCache = new LRUCache<string, Account>({
```

Comment:
Some of this stuff could be DRY'd up by creating an abstract base class. This works for now.